rust/core-lib/src/tabs.rs

```diff
@@ -350,6 +405,7 @@ impl Documents {
         use rpc::CoreRequest::*;
         match cmd {
             NewView { file_path } => {
+                eprintln!("xi-core new view");
```

Comment:
Maybe not intended for this commit?

Reply:
Definitely not.  I was debugging stuff & accidentally committed.  Thanks for catching.